### PR TITLE
HOLY JESUS, WHAT IS THAT? WHAT. THE. FUCK. IS. THAT.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
I'd remove the other src'es, but i just want this pr's diff to highlight this, in all it's glory.

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/life.dm [hunk 1]
```diff
@@ -109,7 +109,7 @@
 						if (loc.master.power_equip)
 							if (!istype(T, /turf/space))
 								src << "Alert cancelled. Power has been restored without our assistance."
-								src.aiRestorePowerRoutine = 0
```
Comment (MrStonedOne) [inline code/modules/mob/living/silicon/ai/life.dm]: Got this one right.

File: code/modules/mob/living/silicon/ai/life.dm [hunk 2]
```diff
@@ -109,7 +109,7 @@
 						if (loc.master.power_equip)
 							if (!istype(T, /turf/space))
 								src << "Alert cancelled. Power has been restored without our assistance."
-								src.aiRestorePowerRoutine = 0
+								aiRestorePowerRoutine = 0
```
Comment (MrStonedOne) [inline code/modules/mob/living/silicon/ai/life.dm]: Got this one right.

**Comment by Jordie0608** [comment]
Flawless

**Comment by duncathan** [comment]
jesus help us all

**Comment by Shapsy** [comment]
I just woke up and I thought this PR was going to have something to do with jelly donuts because of the name

**Comment by Shadowlight213** [comment]
All those colons...
Holy shit.

**Pull request merged**